components/Buttons/PlayButton.vue

```diff
@@ -43,32 +47,57 @@ export default Vue.extend({
       type: Array as () => BaseItemDto[],
       required: true
     },
+    disabled: {
```

Comment:
I don't see a case where we would need to disable this manually. It either can play items, or it can't.

Do you have any example of where and why we might use this?

Reply:
On library view while it's loading, like that old global-playback buttons

Reply:
It's already disabled while loading, since we pass it the items directly (in order to only play what's shown, even with filters on). See: https://github.com/jellyfin/jellyfin-vue/blob/master/pages/library/_viewId.vue#L37-L38